engine/src/io/data_parser/OrcParser.cpp

```diff
@@ -77,7 +77,7 @@ std::unique_ptr<ral::frame::BlazingTable> orc_parser::parse_batch(
 			orc_args.columns[column_i] = schema.get_name(column_indices[column_i]);
 		}
 
-		orc_args.stripe = stripe;
+		orc_args.stripe_list = row_groups;
```

Comment:
I dont think this is what we want. Right now, we are not collecting metadata to actually know which or how many stripes to read. Here, we would have the vector of row_groups have one element with -1, which i dont think will be an appropriate argument to stripe_list. I think we should change this line to this:
```
if (row_groups.size() == 0 || (row_groups.size() == 1 && row_groups[0] == -1)){
    orc_args.stripe = -1;  // load all
} else {
    orc_args.stripe_list = row_groups;
}
```